eth/tracers/live/tests/supply/eip1559_burn.json

```diff
@@ -0,0 +1,87 @@
+{
```

Comment:
All these json-files should be placed in a `testdata` folder, instead. E.g. `testdata/supply` or just `testdata`